test/unit_tests/unit_test_image_function.cpp

```diff
@@ -65,222 +65,12 @@ namespace
     };
 }
 
-namespace Function_Template
+
+namespace Function_Template_Unit_Test
```

Comment:
What was a reason to rename this namespace?

Reply:
Oh, the change was to help distinguish function template namespace for unit tests from perfomance tests... Do you want me to revert it?

Reply:
Yes, please, except my change for Unit_Test namespace. Otherwise I can do myself little later if you are busy :)

Reply:
Reverted! If there anything else you feel should be changed - do tell :)

Reply:
Everything looks fine. Thanks :)